RiotNSE/NotificationService.swift

```diff
@@ -471,8 +471,12 @@ class NotificationService: UNNotificationServiceExtension {
                                 notificationBody = NotificationService.localizedString(forKey: "VIDEO_FROM_USER", eventSenderName)
                             case kMXMessageTypeAudio:
                                 if event.isVoiceMessage() {
-                                    // Ignore voice broadcast chunk event
-                                    if event.content[VoiceBroadcastSettings.voiceBroadcastContentKeyChunkType] == nil {
+                                    // Ignore voice broadcast chunk event except the first one.
+                                    if let chunkInfo = event.content[VoiceBroadcastSettings.voiceBroadcastContentKeyChunkType] as? [String: UInt] {
+                                        if chunkInfo[VoiceBroadcastSettings.voiceBroadcastContentKeyChunkSequence] == 1 {
+                                            notificationBody = NotificationService.localizedString(forKey: "VOICE_BROADCAST_FROM_USER", eventSenderName)
```

Comment:
@yostyle it seems that this localization key VOICE_BROADCAST_FROM_USER is missing

Reply:
You right. I forgot to push this update. It's fixed now.